arviz/plots/tsplot.py

```diff
@@ -0,0 +1,430 @@
+"""Plot timeseries data."""
+import warnings
+import numpy as np
+
+from ..sel_utils import xarray_var_iter
+from ..rcparams import rcParams
+from .plot_utils import default_grid, get_plotting_function
+
+
+def plot_ts(
+    idata,
+    y,
+    x=None,
+    y_hat=None,
+    y_holdout=None,
+    y_forecasts=None,
+    x_holdout=None,
+    plot_dim=None,
+    holdout_dim=None,
+    num_samples=100,
+    backend=None,
+    backend_kwargs=None,
+    y_kwargs=None,
+    y_hat_plot_kwargs=None,
+    y_mean_plot_kwargs=None,
+    vline_kwargs=None,
+    textsize=None,
+    figsize=None,
+    legend=True,
+    grid=False,
+    axes=None,
+    show=None,
+):
+    """Plot timeseries and it's components.
+
+    Parameters
+    ----------
+    idata : InferenceData
+    y : str, variable name from observed_data
```

Comment:
```suggestion
    y : str
        variable name from observed_data
```

the "first line" should have only the argument name, type info and optional/default info. Any extra description should go in the "second line/paragraph" that is indented.

Reply:
Oh.. sure!